### PR TITLE
docs: updated PolygonParts examples and profile(MAPCO-5170)

### PR DESCRIPTION
**Pull request by RonitKissis**

### CODE REVIEW

**Review by shimoncohen — CHANGES_REQUESTED** [review]

File: docs/MapColonies/Raster/services/PolygonParts/README.md [hunk 1]
```diff
@@ -151,25 +137,25 @@ To list all the available feature types use the `GetCapabilities` operation and
...
 
-We got a `FeatureTypeList` consisting of the different `FeatureTypes` that each one hold and represent some catalog layer under the WFS service. one of them is the `polygon_parts:ORTHOPHOTO_MOSAIC_BASE-OrthophotoBest`.`FeatureType` where polygon_parts is the namespace and the orthophoto_mosaic_base_ORTHOPHOTO_MOSAIC_BASE is the unique `FeatureType` name under the polygon_parts namespace - it include layer's `product_name` + `product_type`. it is possible to query the FeatureType by its full name `polygon_parts:ORTHOPHOTO_MOSAIC_BASE-OrthophotoBest` or short unique name `ORTHOPHOTO_MOSAIC_BASE-OrthophotoBest`.
+We got a `FeatureTypeList` consisting of the different `FeatureTypes` that each one hold and represent some catalog layer under the WFS service. one of them is the `polygonParts:ORTHOPHOTO_BEST-OrthophotoBest`.`FeatureType` where polygonParts is the namespace and the ORTHOPHOTO_BEST-OrthophotoBest is the unique `FeatureType` name under the polygonParts namespace - it include layer's `productId` + `productType`. it is possible to query the FeatureType by its full name `polygonParts:ORTHOPHOTO_BEST-OrthophotoBest` or short unique name `ORTHOPHOTO_BEST-OrthophotoBest`.
```
[the remark this answers is not in the document]
Comment (RonitKissis) [reply docs/MapColonies/Raster/services/PolygonParts/README.md:140]: yes. We updated the wfs layer in prod azure env and wanted to update all the requests and response according to it, to enable easy and smooth integration with getMap as requested in the ticket.

File: docs/MapColonies/Raster/services/PolygonParts/README.md [hunk 2]
```diff
@@ -370,141 +356,273 @@ Now that we hold the structure of the `orthophoto_mosaic_base_ORTHOPHOTO_MOSAIC_
...
+                "imagingTimeBeginUtc": "2022-08-20T23:08:10Z",
+                "imagingTimeEndUtc": "2022-08-20T23:08:10Z",
+                "resolutionDegree": 0.00000536441802978516,
+                "resolutionMeter": 0.6,
+                "sourceResolutionMeter": 0.1,
+                "horizontalAccuracyCe90": 1.5,
+                "sensors": "ישראל, מצרים, ירדן, לבנון, סוריה, ערב הסעודית",
+                "countries": "עולמי",
+                "cities": "",
+                "description": "תשתית אורתופוטו באיזור צפון עזה עדכנית לאוגוסט 2022"
```
[the remark this answers is not in the document]
Comment (RonitKissis) [reply docs/MapColonies/Raster/services/PolygonParts/README.md:600]: This is the real data stored in the DB and represents the layer parts

File: docs/MapColonies/Raster/services/PolygonParts/README.md [hunk 3]
```diff
@@ -535,168 +657,106 @@ We'll invoke a POST GetFeature request with the following body:
...
+                "imagingTimeBeginUtc": "2021-06-11T02:00:00Z",
+                "imagingTimeEndUtc": "2021-06-11T02:00:00Z",
+                "resolutionDegree": 0.00000536441802978516,
+                "resolutionMeter": 0.7,
+                "sourceResolutionMeter": 0.07,
+                "horizontalAccuracyCe90": 4,
+                "sensors": "OTHER",
+                "countries": "",
+                "cities": "",
+                "description": "תשתית אורתופוטו באיו\"ש עדכני ליוני 2021"
```
[the remark this answers is not in the document]
Comment (RonitKissis) [reply docs/MapColonies/Raster/services/PolygonParts/README.md:711]: Same here.

File: docs/MapColonies/Raster/services/PolygonParts/README.md [hunk 4]
```diff
@@ -706,339 +766,584 @@ We'll invoke a POST GetFeature request with the following body:
...
+                "imagingTimeBeginUtc": "2022-08-20T23:08:10Z",
+                "imagingTimeEndUtc": "2022-08-20T23:08:10Z",
+                "resolutionDegree": 0.00000536441802978516,
+                "resolutionMeter": 0.6,
+                "sourceResolutionMeter": 0.1,
+                "horizontalAccuracyCe90": 1.5,
+                "sensors": "ישראל, מצרים, ירדן, לבנון, סוריה, ערב הסעודית",
+                "countries": "עולמי",
+                "cities": "",
+                "description": "תשתית אורתופוטו באיזור צפון עזה עדכנית לאוגוסט 2022"
```
[the remark this answers is not in the document]
Comment (RonitKissis) [reply docs/MapColonies/Raster/services/PolygonParts/README.md:1010]: Same here.

File: docs/MapColonies/Raster/services/PolygonParts/README.md [hunk 5]
```diff
@@ -706,339 +766,584 @@ We'll invoke a POST GetFeature request with the following body:
...
+                "imagingTimeBeginUtc": "2021-06-11T02:00:00Z",
+                "imagingTimeEndUtc": "2021-06-11T02:00:00Z",
+                "resolutionDegree": 0.00000536441802978516,
+                "resolutionMeter": 0.7,
+                "sourceResolutionMeter": 0.07,
+                "horizontalAccuracyCe90": 4,
+                "sensors": "OTHER",
+                "countries": "",
+                "cities": "",
+                "description": "תשתית אורתופוטו באיו\"ש עדכני ליוני 2021"
```
[the remark this answers is not in the document]
Comment (RonitKissis) [reply docs/MapColonies/Raster/services/PolygonParts/README.md:1126]: Same here.

File: docs/MapColonies/Raster/services/PolygonParts/README.md [hunk 6]
```diff
@@ -706,339 +766,584 @@ We'll invoke a POST GetFeature request with the following body:
...
+                "imagingTimeBeginUtc": "2021-05-12T04:00:00Z",
+                "imagingTimeEndUtc": "2021-05-12T04:00:00Z",
+                "resolutionDegree": 0.00000536441802978516,
+                "resolutionMeter": 0.7,
+                "sourceResolutionMeter": 0.07,
+                "horizontalAccuracyCe90": 4,
+                "sensors": "OTHER",
+                "countries": "",
+                "cities": "",
+                "description": "תשתית אורתופוטו באיו\"ש עדכני למאי 2021"
```
[the remark this answers is not in the document]
Comment (RonitKissis) [reply docs/MapColonies/Raster/services/PolygonParts/README.md:1184]: Same here.

File: docs/MapColonies/Raster/services/PolygonParts/README.md [hunk 7]
```diff
@@ -706,339 +766,584 @@ We'll invoke a POST GetFeature request with the following body:
...
 
 ### Query by geographical limits
 
-For more complex criteria such as a set of multiple parameters or geographical intersections we should invoke a POST GetFetures request consisting the filter as a XML body.
+For more complex criteria such as a set of multiple parameters or geographical intersections we should invoke a **POST** GetFetures request consisting the filter as a XML body.
 
-3. let's look for all the polygon parts features that intersect in a polygon, one of the properties of a polygon part feature is it's geometry describing the polygon part geography, we'll look by it by setting it as the request `ValueReference`. We can specify the `srsName` which is the coordinate reference system of the returned features in our case `EPSG:4326`.
+3. let's look for all the polygon parts features that intersect in a polygon, one of the properties of a polygon part feature is it's `footprint` describing the polygon part geography, we'll look by it by setting it as the request `ValueReference`. We can specify the `srsName` which is the coordinate reference system of the returned features in our case `EPSG:4326`.
    the polygon is a list of latitude-longitude pair coordinates.
```
Comment (CL-SHLOMIKONCHA) [inline docs/MapColonies/Raster/services/PolygonParts/README.md:1336]: longitude-latitude

**Review by CL-SHLOMIKONCHA — CHANGES_REQUESTED** [review]

**Comment by CL-SHLOMIKONCHA** [comment]
sensors responses includes regions names, please change to "OTHER" in all response examples

**Comment by CL-SHLOMIKONCHA** [comment]
Please get rid of description in response examples, leave empty string

**Review by CL-SHLOMIKONCHA — APPROVED** [review]

File: docs/MapColonies/Raster/services/PolygonParts/README.md [hunk 8]
```diff
@@ -594,34 +594,34 @@ Now that we hold the structure of the `ORTHOPHOTO_BEST-OrthophotoBest` FeatureTy
                 "resolutionMeter": 0.6,
                 "sourceResolutionMeter": 0.1,
                 "horizontalAccuracyCe90": 1.5,
-                "sensors": "ישראל, מצרים, ירדן, לבנון, סוריה, ערב הסעודית",
+                "sensors": "OTHER",
                 "countries": "עולמי",
```
Comment (shimoncohen) [inline docs/MapColonies/Raster/services/PolygonParts/README.md:598]: Please change.

**Review by shimoncohen — APPROVED** [review]

**Pull request merged**